core/Files.h

```diff
@@ -99,11 +100,15 @@ class File final {
     /** Given a 1-based line number, returns a string view of the line. */
     std::string_view getLine(int i);
 
+    void setFileHash(std::shared_ptr<FileHash> hash);
```

Comment:
The original version
https://github.com/sorbet/sorbet/commit/9cbe0b084b88ca540ecc3883caf5272ec6514216
 intentionally had `unique_ptr<FileHash>` here. and `shared_ptr<const FileHash>` in reader.
Why was it changed? These are important invariants around how these could be mutated.

Reply:
I was on an airplane and lacked internet or the original diff, so I improvised. This is an easy change to make.